ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -71,10 +139,9 @@ const TWO_POINTS = 6;
         }
 
         // Render data into three.js objects
-        var materials1 = new THREE.LineBasicMaterial( { color: 0xff0000 } );
-        var materials = [new THREE.LineBasicMaterial( { color: 0x00ffff } )];
-        for (var i = 0; i < deserializedData.visualization.length-1; i++) {
-            materials.push(materials1);
+        var materials = [];
+        for (var i = 0; i < structure.visualization.length-1; i++) {
+            materials.push(new THREE.LineBasicMaterial( { color: Colors.random() } ));
```

Comment:
wouldn't this create a new LineBasicMaterial for each line? can we share references? e.g. if I wanted all 100 lines to be red, could I just create one LineBasicMaterial for red and reuse or do i need to create 100 LineBasicMaterial objects?

Reply:
This current setup creates a color for each layer in structure.visualization. So all objects (either lineLoop or lineSegments) will be using the same material.